tests/prefetch.cpp

```diff
@@ -14,16 +15,25 @@
 #include "allocators/collective_allocator.hpp"
 #include "allocators/null_lock.hpp"
 #include "backend/backend.hpp"
+#include "env/env.hpp"
 #include "gtest/gtest.h"
 
 /** @brief ArgoDSM memory size */
 constexpr std::size_t size = 1<<30;
 /** @brief ArgoDSM cache size */
 constexpr std::size_t cache_size = size/8;
 
+namespace env = argo::env;
+namespace dd = argo::data_distribution;
 namespace mem = argo::mempools;
 extern mem::global_memory_pool<>* default_global_mempool;
 
+/** @brief ArgoDSM page size */
+const std::size_t page_size = 4096;
```

Comment:
this is now the 7th or so place where we define this constant. Maybe we should open a ticket to track all of them for a code cleanup PR.

Reply:
#49 already exists